app/assets/javascripts/components/my_lessons_page.es6.jsx

```diff
@@ -0,0 +1,62 @@
+class MyLessonsPage extends React.Component {
+
+  constructor() {
+    super();
+    this.state = {
+      filter: 'upcoming',
+    }
+  }
+
+  handleClick(filter) {
+   this.setState({ filter: filter });
+  }
+
+  renderOption(option) {
+    switch (option) {
+      case 'upcoming':
+        style = (this.state.filter == 'upcoming' ?
+                "button button--lg button--solid-orange" :
+                "button button--lg button--solid-white"
+                );
+        buttonText = 'Upcoming Lessons';
+        break;
+      case 'recent':
+        style = (this.state.filter == 'recent' ?
+                "button button--lg button--solid-orange" :
+                "button button--lg button--solid-white"
+                );
+        buttonText = 'Recent Lessons';
+        break;
+    }
+    return (
+      <Button
+        className={style}
+        onClick={() => this.handleClick(option)}>
+        {buttonText}
+      </Button>
+    );
+  }
+
+  render() {
+    return (
+     <div className="page-wrapper">
+      <Header />
+      <div className="my-lessons-page content-wrapper">
+        <div className="content-container">
```

Comment:
you probably want to put my-lessons-page after content-wrapper because you want your attributes to override the original styling